src/cli/index.js

```diff
@@ -171,6 +181,11 @@ export function main({
   //
   const run = (): Promise<void> => {
     invariant(command, 'missing command');
+
+    if (warnAboutRunDashDash) {
+      reporter.warn(reporter.lang('dashDashDeprecation'));
```

Comment:
Eh, fair point. I'd still get certain things "right" from the start but up to you, not gonna block.